lib/flowViewport/sceneIndex/wireframeHighlights/fvpNiInstanceWhSi.cpp

```diff
@@ -64,9 +64,29 @@ HdSceneIndexPrim NiInstanceWhSi::GetHighlightPrim(const SdfPath &selectionPath,
 
     dsEditor.Set(HdInstancedBySchema::GetDefaultLocator(), HdBlockDataSource::New());
 
+    GfMatrix4d instanceXform(1.0);
     HdSceneIndexPrim instancePrim = GetInputSceneIndex()->GetPrim(selectionKey.first);
-    auto instanceXform = HdXformSchema::GetFromParent(instancePrim.dataSource).GetMatrix()->GetTypedValue(0);
-    auto prototypeXform = HdXformSchema::GetFromParent(prim.dataSource).GetMatrix()->GetTypedValue(0);
+    auto instanceMatrixDS = HdXformSchema::GetFromParent(instancePrim.dataSource).GetMatrix();
+    if (instanceMatrixDS) {
+        instanceXform = instanceMatrixDS->GetTypedValue(0);
+    } else {
+        TF_WARN(
+            "Instance Prim %s (selection %s) has no matrix, using identity.",
```

Comment:
Nit-pick: just to make the two warning messages consistent.
```suggestion
            "Instance Prim %s (selection %s) has no xform matrix data source, using identity.",
```

Reply:
Good point! addressed in 091e69e698